src/main/java/com/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.wolox.training.exceptions;
+
+public class BookNotFoundException extends RuntimeException {
+
+    private final ErrorDescription errorDescription = ErrorDescription.BOOK_NOT_FOUND;
```

Comment:
This final variable has to be final, because you are initializing a field at declaration :+1: